.rubocop.yml

```diff
@@ -5,6 +5,23 @@ AllCops:
     - vendor/**/*
     - gemfiles/**/*
   TargetRubyVersion: 2.4
+  NewCops: enable
```

Comment:
RuboCop has a concept of "pending" cops. As in, they will be activated with 1.0. In my own projects at work we didn't have problems with enabling them using this directive and also the `rubocop` gem is locked to 0.82.0.